changelogs/fragments/73-topic-authorization.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - rabbitmq_user - add support for `topic authorization <https://www.rabbitmq.com/access-control.html#topic-authorisation>`_ (featured in RabbitMQ 3.7.0).
```

Comment:
```suggestion
  - rabbitmq_user - add support for `topic authorization <https://www.rabbitmq.com/access-control.html#topic-authorisation>`_ (featured in RabbitMQ 3.7.0) (https://github.com/ansible-collections/community.rabbitmq/pull/73).
```

Reply:
Very small thing (we usually put issue links but if there isn't we put PR links).

Reply:
that's it from me, I promise:)